package.json

```diff
@@ -48,7 +56,8 @@
     "lint-js": "standard && cd spec && standard",
     "lint-cpp": "python ./script/cpplint.py",
     "lint-py": "python ./script/pylint.py",
-    "lint-docs": "npm run lint-js-in-markdown && npm run create-typescript-definitions",
+    "lint-docs": "npm run lint-docs-prettier && npm run lint-js-in-markdown -- --fix && npm run create-typescript-definitions",
```

Comment:
If the prettier is only being run as part of the `lint-docs` command, the formatted docs won't actually be committed into the repo. Same goes for the `--fix` flag with the `lint-js-in-markdown`. Perhaps do it as a precommit hook?